### PR TITLE
[VBLOCKS-1436] Isolate active call hooks

### PR DESCRIPTION
**Pull request by mhuynh5757**
Isolate active call hooks from components.

https://issues.corp.twilio.com/browse/VBLOCKS-1436

**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

File: app/src/screens/Call/hooks.ts [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+ * @returns - Handlers and state for the dialpad.
+ */
+const useCallDialpad = (
+  activeCall: ActiveCall,
+  dispatch: ReturnType<typeof useTypedDispatch>,
+) => {
+  const [isVisible, setIsVisible] = React.useState<boolean>(false);
+
+  const [handle, isDisabled] = React.useMemo(() => {
+    const noOp = () => {};
```
Comment (charliesantos) [inline app/src/screens/Call/hooks.ts:35]: `noop` function is duplicated multiple times in this file. What do you think of putting this at the top and reusing it?

**Review by charliesantos — CHANGES_REQUESTED** [review]
Looks really good @mhuynh5757 . Left one comment.

**Review by charliesantos — APPROVED** [review]

**Comment by charliesantos** [comment]
@mhuynh5757 thoughts on when we should start writing tests?

**Comment by kbagchiGWC** [comment]
@mhuynh5757 - any thoughts on this? 
> https://github.com/twilio/twilio-voice-react-native-app/pull/11#issuecomment-1428697870

**Pull request merged**